src/main/java/orangehrm/OrangeRecruitmentPage.java

```diff
@@ -0,0 +1,246 @@
+                .findFirst().orElse(null);
+        return this;
+    }
+
+    public OrangeRecruitmentPage toCandidates(){
+        candidates.click();
+        return this;
+    }
+
+    public OrangeRecruitmentPage setName(String candidateName){
+        firstName.sendKeys(candidateName);
+        return this;
+    }
+
+    public OrangeRecruitmentPage setLastName(String candidateLastName){
+        lastName.sendKeys(candidateLastName);
+        return this;
+    }
+
+    public OrangeRecruitmentPage setEmail(String candidateEmail){
+        email.sendKeys(candidateEmail);
+        return this;
+    }
+
+    public OrangeRecruitmentPage setContact(String contactNumber){
+        contactNo.sendKeys(contactNumber);
+        return this;
+    }
+
+    public OrangeRecruitmentPage selectVacancyByName(String title){
+        Select availableVacancies = new Select(candidateVacanciesList);
+        availableVacancies.selectByVisibleText(title);
+        return this;
+    }
+
+    public OrangeRecruitmentPage addComment(String commentText){
+        candidateComment.sendKeys(commentText);
+        return this;
+    }
+
+    public OrangeRecruitmentPage setApplicationDate(String date){
+        appliedDate.clear();
+        appliedDate.sendKeys(date + Keys.ENTER);
+        return this;
+    }
+
+    public OrangeRecruitmentPage allowKeepData(){
+        consentDataCheckbox.click();
+        return this;
+    }
+
+    public OrangeRecruitmentPage applyCandidate(String candidateName, String candidateLastName, String candidateMail,
+                                                String contactNumber, String title, String commentText, String date){
+        return clickAddBtn()
+                .setName(candidateName)
+                .setLastName(candidateLastName)
+                .setEmail(candidateMail)
+                .setContact(contactNumber)
+                .selectVacancyByName(title)
+                .addComment(commentText)
+                .allowKeepData()
+                .setApplicationDate(date)
+                .clickSaveBtn();
+    }
+
+    public OrangeRecruitmentPage findCreatedCandidate(String candidateName, String candidateLastName,
+                                                      String hiringManager, String recordVacancyName, String date){
+        String candidate = String.format("%s %s", candidateName, candidateLastName);
+        createdCandidate = resultList.stream()
+                .filter(o -> o.findElements(By.xpath("./td")).get(1).getText().contains(recordVacancyName) &&
+                        o.findElements(By.xpath("./td")).get(2).getText().contains(candidate) &&
+                        o.findElements(By.xpath("./td")).get(3).getText().contains(hiringManager) &&
+                        o.findElements(By.xpath("./td")).get(4).getText().contains(date))
+                .findFirst().orElse(null);
+        return this;
+    }
```

Comment:
Needs to re-write this method for returning boolean or some value instead of PageObject class.

Reply:
fixed